flexport/latitude
# LDS-444 Do Not Use a Tag for Nonlink Breadcrumb Items

## Description:
Use a `div` instead of an `a` for non-link breadcrumbs. Semantic benefits and avoids potential link styling collisions from before CSS in JS times.

<img width="641" alt="Screen Shot 2019-10-30 at 2 16 38 PM" src="https://user-images.githubusercontent.com/7979583/67900569-056cff80-fb22-11e9-8a5e-35a74937eac7.png">

## Jira:
[LDS-443](https://flexport.atlassian.net/browse/LDS-443)

## Manual Testing Plan:
1. Open Storybook.
2. Confirm Breadcrumb links work correctly and the last breadcrumb does not have a `pointer` cursor or hover effect.
